CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#82](https://github.com/ruby-grape/grape-swagger-entity/pull/82): Fix: recognize grape::api::boolean as a primitive type - [@numbata](https://github.com/numbata).
```

Comment:
```suggestion
* [#82](https://github.com/ruby-grape/grape-swagger-entity/pull/82): Fix: recognize `Grape::API::Boolean` as a primitive type - [@numbata](https://github.com/numbata).
```

Reply:
fixed in [fa7f11a](https://github.com/ruby-grape/grape-swagger-entity/pull/82/commits/fa7f11a5550a50a59c932489d56e2d08c8a9db4c)